db/mongo/client.go

```diff
@@ -47,47 +46,46 @@ func (c client) SaveEmployee(employee *models.Employee) (string, error) {
 	opts := options.Update().SetUpsert(true)
 	collection := c.conn.Database(viper.GetString(config.DbName)).Collection(collectionName)
 	if _, err := collection.UpdateOne(context.TODO(), bson.M{"id": employee.ID}, bson.M{"$set": employee}, opts); err != nil {
-		panic("oops cannot update employee")
+		return "", errors.Wrap(err, "failed to add/update employee")
 	}
-	return "", nil
+	return employee.ID, nil
```

Comment:
Weren't we covering this with unit tests?

Reply:
yes sir

Reply:
we are not covering employee.ID in unit tests